io.github.prateekmedia.appimagepool.yml

```
@@ -0,0 +1,56 @@
+app-id: io.github.prateekmedia.appimagepool
+runtime: org.gnome.Platform
+runtime-version: "40"
+sdk: org.gnome.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.rust-stable
+rename-icon: appimagepool
+rename-appdata-file: appimagepool.appdata.xml
+rename-desktop-file: appimagepool.desktop
+command: appimagepool
+separate-locales: false
+finish-args:
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --socket=pulseaudio
+  - --share=network
+  - --device=dri
+  - --persist=Applications
+  - --talk-name=org.freedesktop.Notifications
+build-options:
+  append-path: /usr/lib/sdk/rust-stable/bin:/app/lib/sdk/flutter/bin
+  env:
+    CARGO_HOME: /run/build/appimagepool/cargo
+modules:
+  - name: flutter
+    buildsystem: simple
+    sources:
+      - type: archive
+        url: https://storage.googleapis.com/flutter_infra_release/releases/beta/linux/flutter_linux_2.6.0-5.2.pre-beta.tar.xz
+        sha256: 1a2d82b9a7934d00d680b6a0a9e09697264dd86ba37d984812c73297f33458e2
+    build-commands:
+      - install -d /app/lib/sdk/flutter
+      - cp -rpv * /app/lib/sdk/flutter/
```

```suggestion
      - cp -rpv * .git /app/lib/sdk/flutter/
```

also it's missing this, I forgot. For some reason flutter want the `.git` and `*` doesn't catch it.